straxen/plugins/peak_processing.py

```diff
@@ -324,3 +324,85 @@ def find_n_competing(peaks, windows, fraction):
             n_tot[i] = n_left[i] + np.sum(areas[i + 1:right_i] > threshold)
 
         return n_left, n_tot
+
+
+class PeakVetoTagging(strax.Plugin):
+    """
+    Plugin which tags S1 peaks according to  muon and neutron-vetos.
+    Tagging S2s is does not make sense as they occur with a delay.
+    However, we compute for both S1/S2 the time delay to the closest veto
+    region.
+
+        * untagged: 0
+        * neutron-veto: 1
+        * muon-veto: 2
+        * both vetos: 3
```

Comment:
Would it make sense to make this an int-num like https://github.com/AxFoundation/strax/blob/master/strax/processing/data_reduction.py#L12? That would make it easier not to accidentally mix up the numbers?